CallHistory/CallHistory/CallLogTableViewCell.swift

```diff
@@ -0,0 +1,91 @@
+//
+//  CallLogTableViewCell.swift
+//  CallHistory
+//
+//  Created by Jaesung Lee on 2021/05/11.
+//
+
+import UIKit
+import SendBirdCalls
+
+class CallLogTableViewCell: UITableViewCell {
+    @IBOutlet weak var callTypeImageView: UIImageView!
+    @IBOutlet weak var nicknameLabel: UILabel!
+    @IBOutlet weak var userIdLabel: UILabel!
+    @IBOutlet weak var startedAtLabel: UILabel!
+    @IBOutlet weak var endReasonLabel: UILabel!
+    
+    var callLog: DirectCallLog! {
+        didSet {
+            updateUI()
+        }
+    }
+    
+    static var dateFormatter: DateFormatter {
+        let dateFormatter = DateFormatter()
+        dateFormatter.dateFormat = "YYYY/MM/dd HH:mm"
+        return dateFormatter
+    }
```

Comment:
```suggestion
    static var dateFormatter: DateFormatter = {
        let dateFormatter = DateFormatter()
        dateFormatter.dateFormat = "YYYY/MM/dd HH:mm"
        return dateFormatter
    }()
```
The getter body will run every time the variable `dateFormatter` is called and create new `DateFormatter` unnecessarily.

Reply:
Thanks!